CPPLINT.cfg

```diff
@@ -1,2 +1,2 @@
 set noparent
-filter=-runtime/reference,-runtime/string,-build/c++11,-build/include_subdir,-whitespace/forcolon,-build/namespaces_literals,-readability/check,-build/include_what_you_use,-readability/nolint
+filter=-runtime/reference,-runtime/string,-build/c++11,-build/include_subdir,-whitespace/forcolon,-build/namespaces_literals,-readability/check,-build/include_what_you_use,-readability/nolint,-readability/braces
```

Comment:
Got this after both concept declarations:
```
/home/szaszm/nifi-minifi-cpp-3/libminifi/include/utils/StringUtils.h:193:  You don't need a ; after a }  [readability/braces] [4]
```

Reply:
@martinzink has an open issue for this for a while now https://github.com/google/styleguide/issues/752

Reply:
To be honest, I have the feeling that we would be better off by dropping cpplint, and rely more on clang-tidy, especially as you recent changes in #1671 get ready and do the google-specific checks.

Reply:
I think it's still worth having cpplint as well, I didn't encounter too many false positives and it has checks that clang tidy does not. It also runs on the Windows codebase while clang tidy only runs on Linux at the moment.